### DISCUSSION TITLE
Github crashes one a specific URL

### DISCUSSION THREAD

**Discussion by mr-cheffy** [community/community#57870] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Navigating to `https://github.com/github.com` by mistake gave me an error on the browser.

**Comment by mrepol742** [comment, marked as answer]
That is because username may only contain alphanumeric characters or single hyphens, and cannot begin or end with a hyphen. And apart from github.com/github.com the github.com/adfeeef.cc or github.com/aege.geg or any random word that does not follow username guidlines will result in 406 tho if you add more than 2 dots like this https://github.com/adadad.adadad.adad itll show `Not Found` instead.

**Reply by orgazmformale123456789** [reply]
Hallo

**Reply by mr-cheffy** [reply]
They should show the 404 page instead of simply crashing
